controllers/admin/AdminImagesController.php

```diff
@@ -858,4 +955,56 @@ public function processDelete()
 
         return parent::processDelete();
     }
+
+    public function getImageFormatForm(array $fields): array
```

Comment:
```suggestion
    private function getImageFormatForm(array $fields): array
```
If it's only used internally this should be a private method, we should avoid expending the public apis of controllers that would then be considered public APIs to maintain